SDKPackageManager.ps1

```diff
@@ -0,0 +1,31 @@
+param([Parameter(Mandatory=$true)][string]$arch, [Parameter(Mandatory=$true)][string]$config, [Parameter(Mandatory=$true)][string]$task)
+
+$currentLocation = Get-Location
+$packagePath = "$currentLocation\BuildOutput\$config\$arch\"
+
+if(!$task.CompareTo("i"))
+{
+	write-host "installing packages"
+	add-appxpackage "$packagePath\Microsoft.WindowsAppRuntime.Framework\Microsoft.WindowsAppRuntime.Framework.msix"
```

Comment:
It's just a little strange to me that i and u deal with different packages. ie. There is no 1 to 1 mapping between the 2. Why is that?